index.html

```diff
@@ -0,0 +1,172 @@
+                    The Final Season (Part 2), which is scheduled to air in the fall of 2023!
+                    In the new PV, Armin recalls the memories of the three of them when they were young, and the line
+                    "Goodbye Ellen" of Mikasa who was also in the key image is symbolic!
+                </p>
+                <h3>Release date: October 2023</h3>
+                <p>Platform release: crunchyroll - wit studio NHK</p>
+            </div>
+        </section>
+        <section id="section1" class="section1">
+            <div class="cont-ti">
+                <h2>Main Program</h2>
+            </div>
+            <hr class="line">
+            <ul class="card-container">
+                <li>
+                    <div class="card">
+                        <img src="ressources/icons/pr1.png" alt="">
+                        <h3>Exhibition</h3>
+                        <p>Discover the "Latest Releases" in movies and series, find the most popular productions and
+                            receive recommendations.</p>
+                    </div>
+                </li>
+                <li>
+                    <div class="card">
+                        <img src="ressources/icons/pr2.png" alt="">
+                        <h3>Forum</h3>
+                        <p>Participa en "Discusiones generales", analiza episodios, especula sobre tramas, lee reseñas
+                            de anime y comparte fanart y cosplay.</p>
+                    </div>
+                </li>
+                <li>
+                    <div class="card">
+                        <img src="ressources/icons/pr3.png" alt="">
+                        <h3>Network</h3>
+                        <p> Create your profile, send private messages, join groups, check the events calendar and find
+                            friends with similar interests.</p>
+                    </div>
+                </li>
+                <li>
+                    <div class="card">
+                        <img src="ressources/icons/pr4.png" alt="">
+                        <h3>Lecture</h3>
+                        <p>Browse the schedule, meet guest speakers, participate in Q&A sessions, and enjoy live
+                            streams.</p>
+                    </div>
+                </li>
+                <li>
+                    <div class="card">
+                        <img src="ressources/icons/pr5.png" alt="">
+                        <h3>Workshop</h3>
+                        <p>Learn with featured instructors, sign up for specific times, and access additional resources
+                            and materials.</p>
+                    </div>
+                </li>
+            </ul>
+            <div class="cont-des">
+                <h2 class="mo"><a href="#">SEE MORE</a></h2>
+                <button type="button" class="des">Join into the world of anime</button>
+            </div>
+        </section>
+        <section class="section2">
```

Comment:
- [ ] [REQUIRED ] Good job so far your contributor section is loading dynamically :clap:  But the design is a bit different compared to the original design. :call_me_hand:   Please make sure that this section has use the same font, `height`, `width `, and spacing `padding/margin `. If your `background color  is white, please change it  to the color defined on the original design  in order to make your  design similar to the original design :+1: 
 Check the screenshots below :point_down:  to see the differences.

#### Orignal design
![Screenshot from 2022-10-28 18-19-46](https://user-images.githubusercontent.com/6615394/198707921-f81e8d82-3ba1-46d2-868b-a34d04408983.jpg)

#### Your design
![Capture](https://github.com/AndresEspi/animePage/assets/6615394/4fe0e590-6d3e-4562-a1f4-dc945a59069c)

Reply:
Weldone so far with your project, however, the `speakers` section design is really different from the design template given, and just as suggested by the previous reviewer, you are to make sure that this section has use the same font, height, width, and spacing padding/margin. In this case. please ensure this section displays in 2 rows and 3 columns, and also ensure that the picture alignment is similar to the design given, in your design, the picture and the text is the same height which is not supposed to be so and the picture background should be by the side of the picture. Lastly, ensure the background color is white. Please make these corrections before asking for another review so you have a better design 😄 

Design given :
![image](https://github.com/AndresEspi/animePage/assets/20537712/239bd1b5-e8c4-4af6-897f-f4288e49be75)


Your design :
![image](https://github.com/AndresEspi/animePage/assets/20537712/c79a13f0-7aaf-40a4-bc51-debb9d5429cf)